public/app/features/alerting/unified/components/MenuItemPauseRule.tsx

```diff
@@ -33,30 +39,36 @@ const MenuItemPauseRule = ({ rule, onPauseChange }: Props) => {
       return;
     }
     const ruleUid = rule.rulerRule.grafana_alert.uid;
+    const targetGroup = await getGroup({
```

Comment:
If you `.unwrap()` this then you can simplify a lot of the checks below - don't need to optional chain or `&& onPauseChange...`

Reply:
💯